virttest/qemu_vm.py

```diff
@@ -1231,13 +1231,15 @@ def add_sga(devices):
                 return " -device sga"
 
         def add_watchdog(devices, device_type=None, action="reset"):
-            watchdog_cmd = ""
-            if devices.has_option("watchdog"):
-                if device_type:
-                    watchdog_cmd += " -watchdog %s" % device_type
-                watchdog_cmd += " -watchdog-action %s" % action
-
-            return watchdog_cmd
+            watchdog_devs = []
+            parent_bus = None
+            if device_type and devices.has_device(device_type):
+                if devices.is_pci_device(device_type):
+                    parent_bus = pci_bus
+                watchdog_devs.append(QDevice(device_type, parent_bus=parent_bus))
```

Comment:
Not checked but I think this would add bus=xx, addr=xx to the command line, is that what we want?

Reply:
@vivianQizhu yes, this is intentional. Currently, we use `-device watchdog ***` instead of `-watchdog ***`.